### PR TITLE
driver: bcmdhd: port sony mimo

### PR DESCRIPTION
**Pull request by Kali-**
enable full wifi speed on z4 (bcm4356)

### CODE REVIEW

File: drivers/net/wireless/bcmdhd/dhd_linux.c [hunk 1]
```diff
@@ -526,6 +533,12 @@ module_param(disable_proptx, int, 0644);
 module_param_string(firmware_path, firmware_path, MOD_PARAM_PATHLEN, 0660);
 module_param_string(nvram_path, nvram_path, MOD_PARAM_PATHLEN, 0660);
 
+/* Disable VHT(11ac) mode */
+#if !defined(DISABLE_11AC)
```
Comment (kholk) [inline drivers/net/wireless/bcmdhd/dhd_linux.c:537]: Can we remove that? (from phone here) I don't see any reference to this flag

**Comment by humberos** [comment]
seems it is breaking portable hotspot.

**Comment by Kali-** [comment]
on?
i have tested z4t wifi and hotspot is working fine (except i dont have any connection to share :D )

**Comment by humberos** [comment]
oh, I found the hotspot issue.

https://github.com/sonyxperiadev/device-sony-karin_windy/commit/1095c48d88ef92a0f41f59f1fc2f8ae8707e255e

**Comment by humberos** [comment]
tested on Karin (LTE).

**Comment by humberos** [comment]
@Kali- 
Could you share the stock Z4-WiFi bcmdhd.cal file.
I will compare it with LTE variant.

**Pull request closed without merging**